### PR TITLE
SF Transit Both Bart and Muni Applet for Tidbyt

### PR DESCRIPTION
**Pull request by hassoncs**
Displays predicted arrival times for both Muni and Bart at the same time. Now you can quickly decide which stop to head to when you are leaving the house!

Estimates comes from the [NextBus](https://www.nextbus.com) API for [San Francisco Muni](https://www.sfmta.com/) and the [Bart Legacy API](https://www.bart.gov/schedules/developers/api) for [Bart](https://www.bart.gov/).

There are are few configuration options:
* Set both a bart station and muni stop
* Hide arrival times below a certain number of minutes (so you have time to get there)

sf_bart_muni

### CODE REVIEW

**Comment by hassoncs** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by hassoncs** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by hassoncs** [comment]
recheck

File: apps/sfbartmuni/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+id: sf-next-muni
```
Comment (matslina) [inline apps/sfbartmuni/manifest.yaml:2]:
```suggestion
id: sf-bart-muni
```

File: apps/sfbartmuni/sf_bart_muni.star [hunk 2]
```diff
@@ -0,0 +1,451 @@
...
+    "WDUB": "West Dublin/Pleasanton",
+    "WOAK": "West Oakland",
+}
+
+def main(config):
+    configure(config)
+    all_transit_estimates = fetch_all_transit_estimates(config)
+    return render.Root(child = render_all(all_transit_estimates), max_age = MAX_AGE_SECS)
+
+def configure(config):
```
Comment (matslina) [inline apps/sfbartmuni/sf_bart_muni.star:122]: Could you help me understand why you're using the cache for this?

File: apps/sfbartmuni/sf_bart_muni.star [hunk 3]
```diff
@@ -0,0 +1,451 @@
...
+        ],
+    )
+
+def search_muni_stop(pattern):
+    pattern_lower = pattern.lower()
+    muni_api_key = cache.get(MUNI_API_KEY_NAME)
+    if muni_api_key == None:
+        return [schema.Option(display = "Failed. Need muni_api_key", value = 0)]
+    get_all_stops_url = build_muni_get_all_stops_api_url(muni_api_key)
+    all_muni_stops = fetch_data(get_all_stops_url, get_all_stops_url)
```
Comment (matslina) [inline apps/sfbartmuni/sf_bart_muni.star:211]: Would it be possible to use a higher TTL than 60s for this data?

**Review by matslina — CHANGES_REQUESTED** [review]

**Pull request closed without merging**